astropy/time/tests/test_sidereal.py

```diff
@@ -17,6 +18,7 @@
 within_2_seconds = functools.partial(np.allclose, rtol=1.0, atol=2.0 / 3600.0)
 
 
+@skipif_no_docstrings
```

Comment:
Is this skipping a test that was previously being run successfully?

Reply:
The test is marked to be skipped when pytest itself is run with `-OO` (`python -OO -m pytest` or `PYTHONOPTIMIZE=2 pytest`), since it checks for docstrings that won't exist. It'll be run as normal without the flag.

Reply:
with the decorator:
```
(astropy) bert ~/Documents/astropy (support-OO)
$ pytest -q astropy/time/tests/test_sidereal.py::test_doc_string_contains_models
Internet access disabled
.                                                                                                          [100%]
1 passed in 0.06s
(astropy) bert ~/Documents/astropy (support-OO)
$ PYTHONOPTIMIZE=2 pytest -q astropy/time/tests/test_sidereal.py::test_doc_string_contains_models
Internet access disabled
s                                                                                                          [100%]
1 skipped in 0.05s
```

without the decorator:
```
(astropy) bert ~/Documents/astropy (support-OO)
$ PYTHONOPTIMIZE=2 pytest -q astropy/time/tests/test_sidereal.py::test_doc_string_contains_models
Internet access disabled
F                                                                                                          [100%]
==================================================== FAILURES ====================================================
________________________________________ test_doc_string_contains_models _________________________________________

    def test_doc_string_contains_models():
        """The doc string is formatted; this ensures this remains working."""
        for kind in ("mean", "apparent"):
            for model in SIDEREAL_TIME_MODELS[kind]:
>               assert model in Time.sidereal_time.__doc__
E               TypeError: argument of type 'NoneType' is not iterable

astropy/time/tests/test_sidereal.py:26: TypeError
============================================ short test summary info =============================================
FAILED astropy/time/tests/test_sidereal.py::test_doc_string_contains_models - TypeError: argument of type 'NoneType' is not iterable
1 failed in 0.13s
```